.circleci/config.yml

```diff
@@ -0,0 +1,217 @@
+version: 2
+workflows:
+  version: 2
+  tests:
+    jobs: &workflow_jobs
+      - node4:
```

Comment:
If this is a brand new repo, lets not support 4.

Reply:
It is brand new. >= 6?

Reply:
Wow that's smart @JustinBeckwith :)
Need to remove node4 tasks from CirlceCI config as well, and from branch protection settings also. Cool cool cool cool!!111

Reply:
@crwilcox Yup! Node.js 4 is officially EOL, so we don't need to support it with new leaf node libraries.  Less work down the road :)